theurgy.lic

```diff
@@ -120,10 +321,39 @@ get_from_container('wine')
 fput 'pour wine on altar'
 put_in_container('wine')
 
+if DRSkill.getxp('Theurgy') <= 24
```

Comment:
this got changed to:

```
if DRSkill.getxp('Theurgy') <= 24 && DRStats.circle > 60
  comm_truff
elsif DRSkill.getxp('Theurgy') <= 31 && DRStats.circle > 8
  comm_kert
end
```

to give you an idea what I was going for.  I'm blanking on a good way to do this otherwise but I could manage them elsewhere.